src/test/java/teammates/test/cases/action/InstructorCourseRemindActionTest.java

```diff
@@ -75,6 +76,7 @@ public void testExecuteAndPostProcess() throws Exception {
         };
 
         remindAction = getAction(submissionParams);
+        remindAction.setCurrentUrl("/page/instructorCourseDetailsPage?");
```

Comment:
Avoid using magic strings. Use `Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE`, same goes for the instances below

Reply:
Also, you have not tested that your newest addition from `INSTRUCTOR_STUDENT_LIST_PAGE` works.

Reply:
My apologies. Fixing this and will keep it in mind in the future. 
Can I add test cases to the current test? If so, I will add them for InstructorCourseRemind on the INSTRUCTOR_STUDENT_LIST_PAGE right away.

Reply:
As a general guideline, when a new feature is implemented, you should add tests to test it. So yes.

Reply:
Oh okay. I'm on it.